src/featbin/wav-reverberate-test.cc

```diff
@@ -0,0 +1,143 @@
+// featbin/extract-segments.cc
```

Comment:
@vijayaditya: perhaps it makes sense to put these DSP functions in one of the libraries, either `matrix/` (this is where we have `SplitRadixRealFft`) or `feat/`. @danpovey, what would you recommend? We do not run tests in `*bin/` directories.